.goreleaser.yml

```diff
@@ -120,18 +147,18 @@ dockers:
     - "--label=org.opencontainers.image.revision={{.FullCommit}}"
     - "--label=org.opencontainers.image.version={{.Version}}"
     - "--label=org.opencontainers.image.source={{ .GitURL }}"
-  - ids:
-    - tarianctl
-    goos: linux
-    goarch: amd64
-    image_templates:
-    - '{{ .Env.CONTAINER_REGISTRY }}/tarianctl:{{ if .IsSnapshot }}latest{{ else }}{{ trimprefix .Tag "v" }}{{ end }}'
-    dockerfile: Dockerfile-tarianctl
-    skip_push: false
-    build_flag_templates:
-    - "--pull"
-    - "--label=org.opencontainers.image.created={{.Date}}"
-    - "--label=org.opencontainers.image.title={{.ProjectName}}"
-    - "--label=org.opencontainers.image.revision={{.FullCommit}}"
-    - "--label=org.opencontainers.image.version={{.Version}}"
-    - "--label=org.opencontainers.image.source={{ .GitURL }}"
+  # - ids:
+  #   - tarianctl
```

Comment:
@pratikjagrut is this disabled intentionally? Any context?

Reply:
Yes. Tarianctl is a CLI tool so I think we don't need an image for that. We can only release the binary of it and that should suffice.

Reply:
I added this not too long ago. My intention was to improve the execution time of chart test. My plan was for chart test, we can just use latest container images from the repo, without building local images. Feel free to let me know your feedback.

However, I haven't finished that. Currently in `.github/workflows/charts.yaml`, we need to tarianctl to run the test but we build the whole components. My plan was to docker pull tarianctl and copy the executable from the image to the current directory.